src/main/java/com/toofifty/easyblastfurnace/state/FurnaceState.java

```diff
@@ -59,6 +59,10 @@ public boolean has(int ...itemIds) {
         return getQuantity(itemIds) > 0;
     }
 
+	public boolean hasInventory(int ...itemIds) {return getQuantity(itemIds) > config.barsInDispenser(); }
```

Comment:
Can this be renamed to hasEnoughBars to help with readability.

Reply:
Don't know why I have these two methods that do the same thing, condensing into one named hasEnoughBars